libs/sdk-backend-tiger/src/backend/index.ts

```diff
@@ -389,3 +398,41 @@ function createHeaders(implConfig: TigerBackendConfig, telemetry: TelemetryData)
 function isNotAuthenticatedResponse(err: any): boolean {
     return err?.response?.status === 401;
 }
+
+function retryMechanism(client: AxiosInstance): AxiosInstance {
+    // could use axios-retry https://www.npmjs.com/package/axios-retry
+    axiosRetry(client, {
+        retries: 3,
+        retryDelay: (retryCount) => retryCount * 1000,
+        retryCondition: (error) => {
+            return error?.response?.status === 400;
```

Comment:
Why this one?

Reply:
Was just to have some reproducer locally to try the retrying, since BE is not ready yet. Found this to be the easiest one.